apps/backoffice-v2/public/locales/en/translation.json

```diff
@@ -1 +1,5 @@
-{}
+{
+  "home": {
+    "greeting": "Welcome"
```

Comment:
Why did you add a translation file? I don't think we maintain the internationalization

Reply:
The file already existed. The contributor asked if the greeting component he made can be i18 and I approved it since it is a positive thing if technically should do more. Right now our toasts are 100% i18, in use, and maintained.